lib/smart_proxy_ansible/helper.rb

```diff
@@ -0,0 +1,32 @@
+
+module Proxy
+  module Ansible
+
+    class Helper
+      class << self
+        def config_path(config_line, default)
+            # Default to /etc/ansible/roles if config_line is empty
+            return default if config_line.empty?
+
+            config_line_key = config_line.first.split('=').first.strip
+            # In case of commented roles_path key "#roles_path" or #collections_paths, return default
+            return default if ['#roles_path', '#collections_paths'].include?(config_line_key)
+
+            config_line.first.split('=').last.strip
+          end
+
+          def path_from_config(config_key, default_config_file)
+            File.readlines(default_config_file).select do |line|
+              line =~ /^\s*#{config_key}/
+            end
+          rescue Errno::ENOENT, Errno::EACCES => e
+            logger.debug(e.backtrace)
+            message = "Could not read Ansible config file #{DEFAULT_CONFIG_FILE} - #{e.message}"
+            raise ReadConfigFileException.new(message), message
+          end
+
+
```

Comment:
I do not want to be a rubocop, but the alignment in this file is seriously off (2 vs 4 spaces) and there are way too many empty lines inside class definition.

Reply:
sure I'll correct them :) it is draft